app/views/admin/creator_settings/_form.html.erb

```diff
@@ -44,30 +44,38 @@
                           placeholder: ::Settings::UserExperience.primary_brand_color_hex,
                           class: "crayons-color-selector js-color-field ml-2",
                           required: true %>
-   </div>
+    </div>
+  </div>
 </div>
 
 <div class="crayons-field mt-6 align-left">
   <fieldset aria-describedby="section-description">
-    <legend class="fs-l">Who can join this community?</legend>
+    <legend class="crayons-field__label mb-2">Who can join this community?</legend>
```

Comment:
Made the labels consistent with the rest of the labels on the page. In addition added some margin to mimic the designs.